person.go

```diff
@@ -102,71 +102,81 @@ var (
 )
 
 type Person struct {
-	Faker *Faker
+	Faker  *Faker
+	gender string
 }
 
-func (p Person) Suffix() string {
+func (p *Person) Suffix() string {
```

Comment:
Please don't use pointer reference in Generators, we intentionally don't use them.